onnxruntime/core/providers/openvino/backend_utils.cc

```diff
@@ -69,6 +69,39 @@ CreateCNNNetwork(const Provider_ModelProto& model_proto, const GlobalContext& gl
     ORT_THROW(log_tag + "[OpenVINO-EP] Unknown exception while importing model to nGraph Func");
   }
 
+  ValidateCNNNetwork(global_context, subgraph_context, const_outputs_map, ng_function);
+
+  try {
+    return std::make_shared<InferenceEngine::CNNNetwork>(ng_function);
+  } catch (const InferenceEngine::details::InferenceEngineException& e) {
+    ORT_THROW(log_tag + " Exception thrown while making IE::CNNNetwork: " + e.what());
+  } catch (...) {
+    ORT_THROW(log_tag + " Exception thrown while making IE::CNNNetwork");
+  }
+}
+
+InferenceEngine::CNNNetwork CreateCNNNetwork1(const Provider_ModelProto& model_proto, const SubGraphContext& subgraph_context, InferenceEngine::CNNNetwork cnn_network) {
```

Comment:
Could you choose a more descriptive name for the function rather than adding the suffix '1'?

Reply:
Maybe something like CreateCNNNetworkFromBlob

Reply:
I have changed the name to ReadCNNNetwork, since we are using the ReadNetwork() core API within this